amy/extrequests/forms.py

```diff
@@ -458,25 +451,16 @@ def validate_member_code(
         self, request: HttpRequest
     ) -> None | dict[str, ValidationError]:
         errors = dict()
-        affiliation = self.cleaned_data.get("member_affiliation")  # yes/no/unsure
         code = self.cleaned_data.get("member_code", "")
-        if affiliation in ["yes", "unsure"] and code:
-            # ensure that code belongs to a membership
-            try:
-                Membership.objects.get(registration_code=code)
-            except Membership.DoesNotExist:
-                errors["member_code"] = ValidationError(
-                    "This code is invalid. "
-                    "Please contact your Member Affiliate to verify your code."
-                )
-        elif affiliation == "yes" and not code:
-            errors["member_code"] = ValidationError(
-                "This field is required if you selected 'Yes' above."
-            )
-        elif affiliation == "no" and code:
-            errors["member_code"] = ValidationError(
-                "This field must be empty if you selected 'No' above."
-            )
+        error_msg = (
+            "This code is invalid. "
+            "Please contact your Member Affiliate to verify your code."
```

Comment:
Is it expected from someone submitting their form to know who "Member Affiliate" is? Can it be confused with someone on the core team?

If so, I'd add an explanation who Member Affiliate is, e.g.
"Please contact your Member Affiliate (usually a person distributing codes in your organization) to verify your code.

Reply:
I think this wording came from the workshops team, @sheraaronhurt what are your thoughts on Piotr's comment here?

Reply:
LGTM